include/casmutils/xtal/coordinate.hpp

```diff
@@ -60,6 +61,9 @@ class Coordinate
     CASM::xtal::Coordinate casm_coord;
 };
 
+// Apply SymOp to Coordinate
+Coordinate operator*(const sym::CartOp& symop, const Coordinate& coordinate);
```

Comment:
I don't think we want to have this here, move it to `xtal/symmetry.hpp` for now.